### PR TITLE
fix: preload key request

### LINKED ISSUE #5572: Improve Site Speed: Preload key requests
According to the Google Page Speed test https://developers.google.com/speed/pagespeed/insights/?url=eventyay.com preloading key requests could improve the speed by 6 seconds. Please improve the speed by implementing the recommended changes as outlined in this article https://web.dev/uses-rel-preload/.

Screenshot from 2020-11-11 10-36-43

### PR DESCRIPTION
**Pull request by Raunaksingh100**
Fixes #5572 

### description-
fixed preload key request

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Raunaksingh100** [comment]
@iamareebjamal can you tell me the file name for this ??

File: app/index.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
   <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
   <link rel="stylesheet" href="{{rootURL}}assets/open-event-frontend.css">
+  <link rel="preload" href="https://eventyay.com/fonts/lato-v17-latin-300.woff2" as="font" crossorigin="anonymous">
+  <link rel="preload" href="https://eventyay.com/fonts/lato-v17-latin-regular.woff2" as="font" crossorigin="anonymous">
```
Comment (iamareebjamal) [inline app/index.html:13]: What is eventyay.com?

Comment (Raunaksingh100) [reply app/index.html:13]: > What is eventyay.com?

@iamareebjamal I copy the address of the error which is showing in preload key request

Code syantx I used-
<link rel="preload" href="Font URL Showing in PageSpeed" as="font" crossorigin="anonymous">

Comment (Raunaksingh100) [reply app/index.html:13]: And I used this syantx code to solved this issue
<link rel="preload" href="Font URL Showing in PageSpeed" as="font" crossorigin="anonymous">

Comment (iamareebjamal) [reply app/index.html:13]: But this is open event project, not eventyay

Comment (Raunaksingh100) [reply app/index.html:13]: > But this is open event project, not eventyay 

I will fixed

**Comment by Raunaksingh100** [comment]
@iamareebjamal by mistake i delte the branch

**Pull request closed without merging**